co2eq/flights/index.js

```diff
@@ -1,11 +1,17 @@
 import { geoDistance } from 'd3-geo'; // todo - add d3-geo in package.json
```

Comment:
The flight model is used by the transportation model, and thus, if you don't separate the models right away, you'll need to update the model version in transportation.js

Reply:
Ok I see, so you would rather make the full change in this PR? I'll start right away

+ Is there an easy way to test the code of the models?

Reply:
Both are fine honestly. It's also fine to just update the transportation.js model version.
Unfortunately there's no simple way to test the code right now.
@madsnedergaard do you have an idea on how to test without running the app in a simulator?

Reply:
Ok I've submitted a proposal, but I am not sure how the app interprets the models so some changes might be needed.

Reply:
Hmm, unfortunately not - but I think it would be great and fairly straight-forward to write some simple unit tests for the models!
Could be a hackday project for next week ;)